addons/adyen_platforms/controllers/adyen_platforms.py

```diff
@@ -1,13 +1,46 @@
-# -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
+import json
+import logging
+from pprint import pformat
+
 from odoo import http
 from odoo.http import request
 
+from odoo.addons.adyen_platforms.util import odoo_payments_proxy_control
+
+_logger = logging.getLogger(__name__)
+
 
 class AdyenPlatformsController(http.Controller):
 
     @http.route('/adyen_platforms/create_account', type='http', auth='user', website=True)
     def adyen_platforms_create_account(self, creation_token):
         request.session['adyen_creation_token'] = creation_token
         return request.redirect('/web?#action=adyen_platforms.adyen_account_action_create')
+
+    @odoo_payments_proxy_control()
+    @http.route('/adyen_platforms/account_notification', type='json', auth='public', csrf=False)
+    def adyen_platforms_notification(self):
+        data = json.loads(request.httprequest.data)
+        _logger.debug('Account notification received: %s', pformat(data))
+
+        account = request.env['adyen.account'].sudo().search([('adyen_uuid', '=', data.get('adyen_uuid'))])
```

Comment:
Is it really a good idea to potentially look for an adyen account with an unset uuid?

Reply:
I've changed to use `data['adyen_uuid']` instead